week1/armstrong.rb

```diff
@@ -7,7 +7,41 @@
 #
 # Your output should be `true` if the number is an Armstrong number
 # and `false` otherwise.
-
-def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
-end
+def pow(a,b)
+    result = 1
+    b.times do
+        result=result*a
+    end
+    return result
+  end
```

Comment:
Ruby's USP is it's elegance and optimisations for programmer's happiness 🙂
For `pow`, you can just do:
```ruby
x**t
```
Which is equivalent to x^t